README.md

```diff
@@ -68,13 +70,19 @@ More information can be found [here](https://github.com/celestiaorg/celestia-nod
 `<node_type>` can be `bridge`, `full` or `light`.
 
 ```sh
-celestia <node_type> init 
+celestia <node_type> init
 ```
 
 ```sh
 celestia <node_type> start
 ```
 
+### Environment variables
+
+| Variable       | Explanation                         | Default value | Required |
+| -------------- | ----------------------------------- | ------------- | -------- |
+| `BOOTSTRAPPER` | Start the node in bootstrapper mode | `false`       | Optional |
```

Comment:
[proposal][non-blocking]: Please create a separate issue in node repo explaining that there are other variables need to be documented 🙏 , like
```
CELESTIA_CUSTOM=<chain-id>:<gen-hash>
```

Reply:
https://github.com/celestiaorg/celestia-node/issues/1896